lldb/test/API/lang/swift/variables/generic_enums/TestSwiftGenericEnums.py

```diff
@@ -83,24 +59,21 @@ def cleanup():
         self.assertTrue(enumvar.GetValue() is None,
                         "static type has a value when it shouldn't")
         enumvar = enumvar.GetDynamicValue(lldb.eDynamicCanRunTarget)
-        self.assertTrue(
-            enumvar.GetValue() == "Some",
-            "dynamic type's value should be Some")
-        self.assertTrue(
-            enumvar.GetSummary() == "3",
+        # FIXME?
+        #self.assertEqual(
+        #    enumvar.GetValue(), "Some",
+        #    "dynamic type's value should be Some")
+        self.assertEqual(
+            enumvar.GetSummary(), "3",
             "Some's summary should be 3")
 
-        self.runCmd("continue")
-        threads = lldbutil.get_threads_stopped_at_breakpoint(
-            process, breakpoint2)
-
-        self.assertTrue(len(threads) == 1)
+        process.Continue()
 
         value = self.get_variable("value")
         lldbutil.check_variable(
             self,
             value,
             use_dynamic=True,
             summary='"Now with Content"',
-            typename='String?')
+            typename='Swift.Optional<Swift.String>')
```

Comment:
what caused this to change? Shouldn't lldb show the sugared form (`String?`)?

Reply:
This test has been XFAILed since 2019, so I don't really know. I would have also expected the sugared string here.

Reply:
I'll file a bug about this.

Reply:
It is in fact `GetTypeName` vs `GetDisplayTypeName`. I am not sure whether we want to show sugared in both, I can see the argument for having the desugared type name in available somehow, either `GetTypeName` or some other (new?) function.